src/components/SplitButton/__stories__/splitButton.stories.mdx

```diff
@@ -344,6 +359,29 @@ When there’s already a primary button in the view, use a secondary split butto
   </Story>
 </Canvas>
 
+### Custom menu
+
+The split button can accept a custom Menu in case there's a need to override the default behavior.
+
+Notice to always include `focusItemIndexOnMount` prop for accessibility reasons when using custom menus.
```

Comment:
We don't have one standard for it, but I like it better when props names are referenced using `<code>` tag. You can try it out, but the decision is up to you

Reply:
We can maybe try to make it through `preview` file - to assign `<code>` tag to replace this `. But it's definitely out of scope here

Reply:
I think they are rendered the same way in mdx (also on md in general I think) (meaning `hello` is converted to `<code>hello</code>`)

Reply:
yep

<img width="456" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/30905362/ed51c735-de12-49c6-9696-eeabb7a5c984">